ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/util/SampleData.java

```diff
@@ -372,8 +373,8 @@ public static CCDCase getCCDCitizenCase(List<CCDCollectionElement<CCDAmountRow>>
             .sotSignerRole("role")
             .externalReferenceNumber("external ref")
             .externalId(UUID.randomUUID().toString())
-            .feeAccountNumber("PBA1234567")
```

Comment:
comment: just wondering why this has been dropped?

Reply:
fee account number is for solicitor case but this one is for citizen